runtime/testing/e2e/a11y.ts

```diff
@@ -0,0 +1,46 @@
+/*jslint node: true */
+'use strict';
+
+const AxeBuilder = require('axe-webdriverjs');
+import { browser } from 'protractor';
+
+export class A11yTest {
```

Comment:
I would suggest we at a minimum prepend `Sky`, so `SkyA11y`.  Even though, I'm wondering if there's a better name?  `SkyAxeTools` maybe?